src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -240,6 +240,54 @@ def test_when_entity_ref_prefix_type_invalid_then_debug_log_printed(
         )
 
 
+manager_capabilities = [
+    (Manager.Capability.kStatefulContexts, ManagerInterface.Capability.kStatefulContexts),
+    (Manager.Capability.kCustomTerminology, ManagerInterface.Capability.kCustomTerminology),
+    (Manager.Capability.kResolution, ManagerInterface.Capability.kResolution),
+    (Manager.Capability.kPublishing, ManagerInterface.Capability.kPublishing),
+    (Manager.Capability.kRelationshipQueries, ManagerInterface.Capability.kRelationshipQueries),
+    (Manager.Capability.kExistenceQueries, ManagerInterface.Capability.kExistenceQueries),
+    (
+        Manager.Capability.kDefaultEntityReferences,
+        ManagerInterface.Capability.kDefaultEntityReferences,
+    ),
+]
+
+
+class Test_Manager_Capability:
+    @pytest.mark.parametrize(
+        "manager_capability,managerinterface_capability", manager_capabilities
+    )
+    def test_values_match_managerinterface(self, manager_capability, managerinterface_capability):
+        assert manager_capability.value == managerinterface_capability.value
+
+
+class Test_Manager_hasCapability:
+    def test_method_defined_in_cpp(self, method_introspector):
+        assert not method_introspector.is_defined_in_python(Manager.hasCapability)
+        assert method_introspector.is_implemented_once(Manager, "hasCapability")
+
+    @pytest.mark.parametrize(
+        "manager_capability,managerinterface_capability", manager_capabilities
+    )
+    @pytest.mark.parametrize("return_value", (True, False))
+    def test_wraps_the_corresponding_method_of_the_held_interface(
```

Comment:
[question] Did we need to check this has converted the enum properly?

Reply:
That's handled by the `hasCapability(manager_capability)` vs `assert_called_once_with(managerinterface_capability)`, I think.